tools/pagerduty-migrator/scripts/add_users_pagerduty_to_grafana.py

```diff
@@ -0,0 +1,44 @@
+import os
+import secrets
+import sys
+import requests
+
+from urllib.parse import urljoin
+from pdpyras import APISession
+
+PAGERDUTY_API_TOKEN = os.environ["PAGERDUTY_API_TOKEN"]
+PATH_USERS_GRAFANA = "/api/admin/users"
+URL_GRAFANA = os.environ["URL_GRAFANA"] # Example: http://localhost:3000
+USERNAME_GRAFANA = os.environ["USERNAME_GRAFANA"]
+PASSWORD_GRAFANA = os.environ["PASSWORD_GRAFANA"]
```

Comment:
Is it possible to create users through Grafana API using auth tokens? I think we should use Grafana token authentication here if it's possible.

Reply:
Unfortunately its not possible.
As shown in the documentation

https://grafana.com/docs/grafana/latest/developers/http_api/admin/#global-users
The only method for using the API, will be basic authentication